src/az390.java

```diff
@@ -10923,6 +11005,196 @@ private void fp_get_hex(){
 				dc_hex = "8" + dc_hex.substring(1);
 			}
 			return;
+
+		} else if (fp_text.toUpperCase().equals("(INF)")){ // #423
+			switch (tz390.fp_type){  // gen (min) hex for tz390.fp_type
+			case 0: // tz390.fp_db_type s1,e11,m52 with assumed 1
+				dc_hex = "7FF0000000000000";
+			    break;
+			case 1: // tz390.fp_dd_type s1,cf5,bxcf8,ccf50
+				dc_hex = "EEEEEEEEEEEEEEEE"; 
+				break;
```

Comment:
1. INF, NAN, QNAN, and SNAN only apply to binary and decimal floating point. See the HLASM Language Reference manual; p156 of the -9 version. It also shows that DMIN is another special value. MAX, MIN and DMIN apply to all 3 floating point types. The az390 code does NOT implement "(DMIN)". Should it?
2. Why all the "EEE...EEE" values? Why not the correct values? John Ehrman's assembler text has tables that show the exact values. For example, the value for INF for DD is "7800000000000000". See Table 325 in Section 33.5 of the book for all the DFP values. See Table 276 of Section 34.2 for BFP values. See Table 238 in Section 33.4.2 for the MAX, MIN and DMIN values for HFP. Note there are no HFP INF, NAN, QNAN or SNAN values.
3. Unrelated, but some of the MIN values for HFP are incorrect and the MIN values for DFP appear to be DMIN values. Please doublecheck all values against the tables mentioned above.
4. Not sure why you added INF,NAN, QNAN and SNAN for HFP. Shouldn't these generate assembly errors?
5. You should replace eachl "EEE...EEE" value with the corresponding correct value.